### DISCUSSION TITLE
Cannot verify that I'm university student on Azure

### DISCUSSION THREAD

**Discussion by whateveriiwant** [microsoft/SATechnicalOnboarding#64] [Questions]
<img width="1637" alt="Screenshot 2023-10-19 at 20 56 47" src="https://github.com/microsoft/SATechnicalOnboarding/assets/80333011/cd3c01af-1918-4b16-95eb-58e416384099">

Hi, I have started technical onboarding course today.
To start this course, I have to create an Azure for Student account.

But I can't verify that I'm university student. 
I used my personal account(jsj9620@outlook.com) and entered my school email(jsj9620@hufs.ac.kr) in sign up form.
It said there is no access to my account, and cannot check that access is in my account normally.

How can I resolve this problem?

**Comment by AnthonyByansi** [comment]
Hello @whateveriiwant,

It appears that the institution domain is currently not listed among the available domains. @SalemTracey will take care of adding your school domain to the list. In the meantime, you can redeem the [Azure credits](https://azure.microsoft.com/free/?WT.mc_id=%3Fwt.mc_id%3Dstudentamb_260352#all-free-services) by signing up for the free $200 credits for 30 days using your credit card. Rest assured, you won't incur any charges during this period!

Thank you for your patience.

**Comment by AnthonyByansi** [comment]
If you assume the role of a Microsoft (MSFT) ambassador for technical onboarding, I strongly advise that you strive to achieve the Alpha milestone. This milestone signifies your readiness to access Azure's monthly credits without relying on your university email address.

**Comment by whateveriiwant** [comment]
I got promoted to alpha MLSA yesterday, so this problem is solved. Thanks!
